data/102/032/341/102032341.geojson

```diff
@@ -15,8 +15,8 @@
     "iso:country":"SG",
     "lbl:latitude":1.366923,
     "lbl:longitude":103.801292,
-    "lbl:max_zoom":13.0,
-    "lbl:min_zoom":6.0,
```

Comment:
I don't have strong opinions about float values vs int values.. cc @nvkelso, I know this has been a topic before.